geomstats/geometry/product_manifold.py

```diff
@@ -16,21 +16,22 @@ class ProductManifold(Manifold):
     the manifolds M_1, ..., M_n need not be the same, nor of
     same dimension, but the list of manifolds needs to be provided.
 
-    By default, a point is represented by an array of shape
-    [n_samples, dim_1 + ... + dim_n_manifolds] where n_manifolds is the number
-    of manifolds in the product. This type of representation is called
-    'vector'.
+    By default, a point is represented by an array of shape:
+    [n_samples, dim_1 + ... + dim_n_manifolds]
+    where n_manifolds is the number of manifolds in the product.
+    This type of representation is called 'vector'.
 
-    Alternatively, a point can be represented by an array of shape
+    Alternatively, a point can be represented by an array of shape:
     [n_samples, n_manifolds, dim] if the n_manifolds have same dimension dim.
     This type of representation is called `matrix`.
 
     Parameters
     ----------
     manifolds : list
-        List of manifolds in the product.
+        list of manifolds in the product
```

Comment:
use previous docstring: upper case and period at the end, to follow numpy docstrings conventions.